lib/i18n-transform.js

```diff
@@ -153,22 +148,12 @@ var _getLanguageFromI18n = function (i18nObj) {
     _extractPreferredLanguage = function (source, i18nArray, acceptedLanguages) {
         var result = null;
```

Comment:
:nitpick: do you think it's worth getting rid of this variable and just returning `null` at the end?